assets/scss/_legacy.scss

```
@@ -0,0 +1,6 @@
+.legacy {
+  margin-top: 6rem;
+  @include media-breakpoint-down(sm) {
+    margin-top: 2rem;
+  }
```

```suggestion
  }
img[alt="Flux v1 Diagram"] {
  width: 100%;
}
```

If there is a more elegant way to solve this I'm all ears.